### PR TITLE
feat: user register congregation

### PR DESCRIPTION
**Pull request by JudeDavis1**
This PR includes:
- `/register` path
- Loading local congregations and displaying them
- Unfinished 'create' button (for now)

After this is merged, we should think about how we should let the user enter their address then call an API to get latitude/longitude.

### CODE REVIEW

**Comment by JudeDavis1** [comment]
Screenshot 2023-11-16 at 17 04 58


image

**Comment by JudeDavis1** [comment]
Yes, I know on desktop it seems a little too spacious on the right. We can add more eye candy or something later. Let me know if you have an idea of what we could fill it up there with.

File: backend/internal/handlers/find_meetings.go [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
 
 	"github.com/gin-gonic/gin"
 )
 
 func FindLocalMeetings(ctx *gin.Context) {
-	latitude := ctx.Query("latitude")
-	longitude := ctx.Query("longitude")
+	var dto dtos.FindLocalMeetingsRequestDTO
+
+	fmt.Println("ALKDSJFALKSDFJ")
```
Comment (j-koziel) [inline backend/internal/handlers/find_meetings.go:15]: little bit of raging I see

**Review by j-koziel — APPROVED** [review]
Good work!

**Comment by j-koziel** [comment]
> .

This is cool

**Comment by j-koziel** [comment]
> Yes, I know on desktop it seems a little too spacious on the right. We can add more eye candy or something later. Let me know if you have an idea of what we could fill it up there with.

Maybe we could just make the list fill up the whole page, and instead of having a register congregation button just use the select buttons to auto create a congregation... I feel like the current implementation would make more sense if the user was selecting multiple items from that list but since they are only making one congregation it makes sense to only select one of them

**Comment by JudeDavis1** [comment]
> > Yes, I know on desktop it seems a little too spacious on the right. We can add more eye candy or something later. Let me know if you have an idea of what we could fill it up there with.
> 
> 
> 
> Maybe we could just make the list fill up the whole page, and instead of having a register congregation button just use the select buttons to auto create a congregation... I feel like the current implementation would make more sense if the user was selecting multiple items from that list but since they are only making one congregation it makes sense to only select one of them

Yeaaa I see what you mean, good point. I just thought of a couple interesting cloud services I saw when selecting single options. We can look into this later

**Pull request merged**